src/pymap3d/ellipsoid.py

```diff
@@ -62,9 +67,17 @@ def __init__(self, model: str = "wgs84"):
             self.semimajor_axis = 1187000.0
             self.semiminor_axis = self.semimajor_axis
         else:
-            raise NotImplementedError(
+            warnings.warn(
                 f"{model} model not implemented, let us know and we will add it (or make a pull request)"
             )
+            return False
+
+        self.get_parameters()
```

Comment:
I'd suggest naming this function:
```suggestion
        self.calculate_parameters()
```
because it's not returning any parameters, just calculating them for later.